public/index.html

```diff
@@ -16,6 +16,6 @@
 	</head>
 	<body>
 		<noscript>You need to enable JavaScript to run this app.</noscript>
-		<div id="root"></div>
+		<main id="root"></div>
```

Comment:
Probably should be:
<main id="root"></main>

Reply:
Ups. :sweat_smile: Good catch!

Reply:
Yes, we know about this, because that is also the reason why automatic checks failed. :sweat_smile:
https://github.com/out-fox-it/aj-ty-spojka/pull/43/checks?check_run_id=1813175779
```suggestion
		<main id="root"></main>
```

Reply:
**_RIP_**

![ibrokeit](https://user-images.githubusercontent.com/76131194/106599377-43dd3f80-6559-11eb-8928-827176581ca2.png)